README.md

```diff
@@ -14,7 +14,7 @@ one-to-one and onto function (bijection) $f: V_1 \rightarrow V_2$ such that $(u,
 
 Bijectivity means that each unique node in graph $A$ can be mapped to a corresponding unique node in graph $B$. This means that there must be the same number of nodes in both graphs. <br><br>
 A graph is completely connected when every pair of distinct vertices is connected by a unique edge. <br><br>
-Consider the edge between distinct nodes $u$ and $v$ in graph $A$. Because A is completely connected, there must be an edge between them. And since both graphs are bijective, $(f(u), f(v))$ must exist in graph $B$. This goes for every vertex in both graphs.<br><br>
+Consider the edge between distinct nodes $u$ and $v$ in graph $A$. Because $A$ is completely connected, there must be an edge between them. And since both graphs are bijective, $(f(u), f(v))$ must exist in graph $B$. This goes for every vertex in both graphs.<br><br>
```

Comment:
As a note, another property of isomorphic graphs is that every corresponding node should have the same degree. That means, there will be an equal amount of degree sequence in the given graphs. So, in the complete graph of n nodes, every node has a degree of n-1.